modules/op-using-a-custom-database-in-tekton-hub.adoc

```diff
@@ -0,0 +1,47 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="op-using-a-custom-database-in-tekton-hub_{context}"]
+= Using a custom database in {tekton-hub}
+
+[role="_abstract"]
+After installation of {tekton-hub} on a cluster, administrators can use a custom database instead of the default PostgreSQL database installed by the Operator.
```

Comment:
```suggestion
As an administrator, after installing {tekton-hub} on a cluster, you can configure it use a custom database instead of the default PostgreSQL database installed by the Operator.
```

Reply:
Can we express _why_ the user might want to do this? What's the benefit of, or use case for, making this change?

Reply:
@PuneetPunamiya can you comment on WHY users might want to make this change?

Reply:
It is because if user wants to use his database and not the default one provided by Hub then in that case user can use his database with `db-migration`, `api` and `ui` provided by Hub